pkg/scheduler/core/generic_scheduler.go

```diff
@@ -93,7 +93,8 @@ func (g *genericScheduler) findClustersThatFit(
 	defer metrics.ScheduleStep(metrics.ScheduleStepFilter, time.Now())
 
 	var out []*clusterv1alpha1.Cluster
-	clusters := clusterInfo.GetReadyClusters()
+	// DO NOT filter unhealthy cluster, let users make decisions by using ClusterTolerations of Placement.
```

Comment:
Hi @likakuli, I think we still need to filter ready clusters. The taint is not added into a cluster immediately when a cluster becomes unhealthy until `FailoverEvictionTimeout` is up, default to 5 minutes.  So in this period of time, we'd better not consider these clusters as schedulable.

Reply:
> The taint is not added into a cluster immediately

I see, so the code to add taint when cluster not healthy is added above the `FailoverEvictionTimeout ` judge.

Reply:
That's right. You could add this taint when reconciling.

Reply:
yeah, make sense, let me do it

Reply:
done. Add a method `taintClusterByCondition` in `syncCluster`.